### PR TITLE
Refactor project to use Babel and ES6

### PR DESCRIPTION
**Pull request by philplckthun**

### CODE REVIEW

**Comment by rgbkrk** [comment]
:sparkles:

File: src/parser.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+};
+
+const COMMANDS = {
+  0: o => o,
+  1: o => Object.assign({}, o, {
+    data: JSON.parse(o.payload)
+  }),
+  2: o => o, // TODO: raise an event, we might want to disconnect
+  3: o => o, // TODO: this needs to be parsed inside the dissolve.loop
+  4: o => o, // TODO: this needs to be parsed inside the dissolve.loop
```
Comment (rgbkrk) [inline src/parser.js]: Are these all the identity function?

Comment (philplckthun) [inline src/parser.js]: Not sure what you mean by identity functions? I just replaced the functions with arrow functions and split the names from them

Comment (rgbkrk) [inline src/parser.js]: f(x) returning x

**Comment by rgbkrk** [comment]
Looking good. Since I know from gitter you want to do follow up to this and only one interface that I can tell has changed (`UDPRelay.listen`), let's merge this and keep going.

**Comment by philplckthun** [comment]
@rgbkrk I also added the relays to the index.js exports btw

**Comment by rgbkrk** [comment]
Cool cool. We'll adapt that in pipboyrelay.

Thanks again for doing this. I always hate reviewing refactor dialect PRs, though I'm really thankful after its done.

**Pull request merged**